migrations/versions/1090_fix_external_ids.py

```diff
@@ -0,0 +1,47 @@
+"""Fix external IDs that were populated into audit events incorrectly
```

Comment:
Real small but might be worth putting in the top line that is audit events regarding direct award projects

Reply:
Sorry, in particular `create_project` audit events

Reply:
Updated.